### ISSUE TITLE
Windows Update installing version 1.0.26 fails with 0x80070643 / Manual install ask for disc

### ISSUE THREAD

**Issue by TimMoser92** [microsoft/wslg#422]
# Environment

```none
Windows build number: 10.0.22000.160
Your Distribution version: Ubuntu 20.04
Your WSLg version: 1.0.24
```

# Steps to reproduce

Use Windows Update to install the latest WSLg Update.
_or_
Use `wsl --update` to install the latest WSLg Update.
_or_
Download the released `msi` file and use that to install the latest update.

**WSL logs**: 

[wsl-logs.zip](https://github.com/microsoft/wslg/files/7026862/wsl-logs.zip)

#  Expected behavior

The update should install without an error code and without asking for any additional input.

# Actual behavior

The installation of the update 1.0.26 is failing since it's release. Windows Update made multiple attempts. 
If I try to install it manually using `wsl --update` it will ask me for an installation source (literally it is asking for a disc) but will not accept the msi of version 1.0.26. Without the source it will fail.

**Comment by usamoi** [comment]
The same problem bothers me too. It asks for a disc when I want to update or uninstall WSL and WSLg.
OS Version: 22000.194

**Comment by TimMoser92** [comment]
Thank you, @usamoi! I used the program you linked to and it suggested an uninstall of WSLg. I followed the suggestion and it solved the problem: after the uninstall I could run `wsl --update` without any problems and now WSLg is up to date.
Maybe a normal uninstall would have helped as well - I do not remember if I tried that before reporting here. Anyways, the problem is gone and I will go ahead and close the issue.
